### DISCUSSION TITLE
differences between cases and files and to assign differents attributes

### DISCUSSION THREAD

**Discussion by Newbie-Alt** [ccbogel/QualCoder#887] [Q&A]
Hello, I'm sorry for my basic question.
I have difficulty organizing cases and files because I don't understand the differences well. I was thinking of creating a case that brings together all the people's interviews and a case for collective meetings. And then have different attributes for each of these cases (for example gender, function, organization... for interviews; and date, program... for meetings). But I have the impression that it doesn't work like that. Could you please give me some ways to organize myself? How can I have different attributes depending on the type of file, interviews or meetings? Is it useful to create cases or should I instead enter each file independently without making a case?
thanks a lot for your help

**Comment by ccbogel** [comment]
Hi. Ultimately all the work is done using the files. So files are the most important. You could use files only without cases, by assigning attributes there. for interviewee id or name, gender etc.

**Reply by Newbie-Alt** [reply]
thank you for your help.
